docs/design/persistentdisks.md

```diff
@@ -0,0 +1,199 @@
+# Persistent Storage
+
+
+## Abstract
+
+This document defines persistent, cluster-scoped storage for applications requiring long lived data.  The term "PersistentStorage" was used to avoid overloading the term "Volume."  Storage is exposed to a pod as a volume.  Developers and pod authors request storage by defining their performance needs and are matched with an appropriate object from a pool maintained by administrators.
+
+Kubernetes makes no guarantees at runtime that these volumes exist or are available. High availability is left to the storage provider.
+
+
+## Goals
+
+* Allow administrators to create and manage pools of persistent storage
+* Allow developers and pod authors to request storage from the pool
+* Implement dynamic storage for cloud providers (using GCE/AWS APIs to create disks/volumes)
+* Provide working examples for each type of storage and cloud provider.
+* Ensure developers and pod authors can rely on storage being available, without being closely bound to a particular disk, server, network topology, or storage technology.
+* Implement  [ #1515 - Eric Tune's durable data](https://github.com/GoogleCloudPlatform/kubernetes/pull/1515) - by maintaining LastMount on PersistentStorage.Status for the pod scheduler to inspect.
+* Use or don't impede [ #2598 - Tim Hockin's volumes framework](https://github.com/GoogleCloudPlatform/kubernetes/pull/2598)
+
+## New Types
+
+_**Detailed Structs in types.go coming next **_
+
+`PersistentStorage` -- new long lived entity that represents a storage request by pod authors.  Contains Spec and Status.
+
+`PersistentStorageSpec` -- a description of the storage being requested
+
+`PersistentStorageStatus` -- represents information about the status of a storage device
+
+`BoundStorage` -- a binding between PooledStorage and a PersistentStorage request.
```

Comment:
@erictune I am already removing local storage from the doc and will have an update shortly.

For clarification, "BoundStorage" was misnamed.  It has nothing to do with pods or where storage is located.  The better name is PersistentStorageBinding which binds available storage from the pool to a PersistentStorage request by a pod author.